Update Teams Shifts Connector Cmdlets to V2 APIs

Shane-Malone opened the pull request.

get-itips commented on teams/teams-ps/teams/Get-CsTeamsShiftsConnection.md:
```diff
@@ -0,0 +1,252 @@
+external help file: Microsoft.Teams.ConfigAPI.Cmdlets-help.xml
+Module Name: MicrosoftTeams
+title: Get-CsTeamsShiftsConnection
+author: shanemalone
```
author should be your GitHub username

get-itips reviewed (changes requested):
Please fix the author metadata as indicated. Thanks

yogkumgit commented:
@Shane-Malone

dariomws commented on teams/teams-ps/teams/New-CsTeamsShiftsConnection.md:
```diff
@@ -0,0 +1,333 @@
+
+## EXAMPLES
+
+### Example WFM 1 Connection
```
```suggestion
### Example 1
```

dariomws commented on teams/teams-ps/teams/New-CsTeamsShiftsConnection.md:
```diff
@@ -0,0 +1,333 @@
+
+* To get the error object and object details: `$result, $result.Detail`
+
+### Example WFM 2 Connection
```
```suggestion
### Example 2
```

dariomws commented on teams/teams-ps/teams/Set-CsTeamsShiftsConnection.md:
```diff
@@ -0,0 +1,393 @@
+
+## EXAMPLES
+
+### Example WFM 1
```
```suggestion
### Example 1
```

dariomws commented on teams/teams-ps/teams/Set-CsTeamsShiftsConnection.md:
```diff
@@ -0,0 +1,393 @@
+
+* To get the error object and object details: `$result, $result.Detail`
+
+### Example WFM 2
```
```suggestion
### Example 2
```

dariomws commented on teams/teams-ps/teams/Set-CsTeamsShiftsConnectionInstance.md:
```diff
@@ -18,163 +18,84 @@ This cmdlet updates a Shifts connection instance.
 
 ## EXAMPLES
 
 ### Example WFM 1
```
```suggestion
### Example 1
```

dariomws reviewed (approved):
Some non-critical suggestions.

Will some of these cmdlets be included in the next PS module release?


Thanks!

Shane-Malone commented on teams/teams-ps/teams/Get-CsTeamsShiftsConnection.md:
```diff
@@ -0,0 +1,252 @@
+## PARAMETERS
+
+### -Break
+Wait for .NET debugger to attach
```
This parameter is provided as is done for all other shifts teams connection cmdlets

JohanFreelancer9 commented on teams/teams-ps/teams/New-CsTeamsShiftsConnectionInstance.md:
````diff
@@ -188,74 +155,153 @@ Accept pipeline input: False
+```
+
+### -SyncScenarioTimeOffRequest
+The sync state for the time off request scenario. Valid values are "Disabled", "FromWfmToShifts" or "TwoWay".
````
```suggestion
The sync state for the time off request scenario. Valid values are "Disabled", "FromWfmToShifts", and "TwoWay".
```

JohanFreelancer9 replied:
```suggestion
The sync state for the time off request scenario. Valid values are "Disabled", "FromWfmToShifts", and "TwoWay".
```

JohanFreelancer9 commented on teams/teams-ps/teams/Set-CsTeamsShiftsConnection.md:
```diff
@@ -0,0 +1,393 @@
+## PARAMETERS
+
+### -Body
+The request body 
```
```suggestion
The request body.
```

JohanFreelancer9 replied:
```suggestion
The request body.
```

JohanFreelancer9 commented on teams/teams-ps/teams/Update-CsTeamsShiftsConnection.md:
````diff
@@ -0,0 +1,329 @@
+```
+
+### -InputObject
+Identity Parameter
````
```suggestion
Identity parameter.
```

JohanFreelancer9 replied:
```suggestion
Identity parameter.
```

JohanFreelancer9 commented on teams/teams-ps/teams/Get-CsTeamsShiftsConnection.md:
````diff
@@ -0,0 +1,252 @@
+```
+
+### -ConnectionId
+The connection id.
````
```suggestion
The connection ID.
```

JohanFreelancer9 commented on teams/teams-ps/teams/New-CsTeamsShiftsConnection.md:
````diff
@@ -0,0 +1,333 @@
+}
+```
+
+Returns the object of created connection.
````
```suggestion
Returns the object of the created connection.
```

JohanFreelancer9 commented on teams/teams-ps/teams/New-CsTeamsShiftsConnection.md:
```diff
@@ -0,0 +1,333 @@
+
+### -Body
+
+The request body
```
```suggestion
The request body.
```

JohanFreelancer9 commented on teams/teams-ps/teams/New-CsTeamsShiftsConnection.md:
````diff
@@ -0,0 +1,333 @@
+```
+
+### -Break
+Wait for .NET debugger to attach
````
```suggestion
Wait for .NET debugger to attach.
```

JohanFreelancer9 commented on teams/teams-ps/teams/New-CsTeamsShiftsConnection.md:
````diff
@@ -0,0 +1,333 @@
+```
+
+### -ConnectorId
+The WFM connector Id.
````
```suggestion
The WFM connector ID.
```

JohanFreelancer9 commented on teams/teams-ps/teams/New-CsTeamsShiftsConnection.md:
````diff
@@ -0,0 +1,333 @@
+```
+
+### -ProxyCredential
+Credentials for a proxy server to use for the remote call
````
```suggestion
Credentials for a proxy server to use for the remote call.
```

JohanFreelancer9 commented on teams/teams-ps/teams/New-CsTeamsShiftsConnection.md:
````diff
@@ -0,0 +1,333 @@
+```
+
+### -ProxyUseDefaultCredentials
+Use the default credentials for the proxy
````
```suggestion
Use the default credentials for the proxy.
```

JohanFreelancer9 commented on teams/teams-ps/teams/New-CsTeamsShiftsConnectionInstance.md:
```diff
@@ -188,74 +155,153 @@ Accept pipeline input: False
-### -ConnectorId
-The connector id
+### -ConnectionId
+Gets or sets the WFM connection id for the new instance. This can be retrieved by running [Get-CsTeamsShiftsConnection](Get-CsTeamsShiftsConnection.md).
```
```suggestion
Gets or sets the WFM connection ID for the new instance. This can be retrieved by running [Get-CsTeamsShiftsConnection](Get-CsTeamsShiftsConnection.md).
```

JohanFreelancer9 commented on teams/teams-ps/teams/New-CsTeamsShiftsConnectionInstance.md:
````diff
@@ -188,74 +155,153 @@ Accept pipeline input: False
+```
 
+### -DesignatedActorId
+Gets or sets the designated actor id that App acts as for Shifts Graph Api calls.
````
```suggestion
Gets or sets the designated actor ID that App acts as for Shifts Graph API calls.
```

JohanFreelancer9 commented on teams/teams-ps/teams/New-CsTeamsShiftsConnectionInstance.md:
```diff
@@ -188,74 +155,153 @@ Accept pipeline input: False
-### -ConnectorSpecificSettings
-The connector specific settings
+### -SyncScenarioOfferShiftRequest
+The sync state for the offer shift request scenario. Valid values are "Disabled", "FromWfmToShifts" or "TwoWay".
```
```suggestion
The sync state for the offer shift request scenario. Valid values are "Disabled", "FromWfmToShifts", and "TwoWay".
```

JohanFreelancer9 commented on teams/teams-ps/teams/New-CsTeamsShiftsConnectionInstance.md:
````diff
@@ -188,74 +155,153 @@ Accept pipeline input: False
+```
 
+### -SyncScenarioOpenShift
+The sync state for the open shift scenario. Valid values are "Disabled", "FromWfmToShifts" or "TwoWay".
````
```suggestion
The sync state for the open shift scenario. Valid values are "Disabled", "FromWfmToShifts", and "TwoWay".
```

JohanFreelancer9 commented on teams/teams-ps/teams/New-CsTeamsShiftsConnectionInstance.md:
```diff
@@ -188,74 +155,153 @@ Accept pipeline input: False
-### -DesignatedActorId
-Gets or sets the designated actor id that App acts as for Shifts Graph Api calls.
+### -SyncScenarioOpenShiftRequest
+The sync state for the open shift request scenario. Valid values are "Disabled", "FromWfmToShifts" or "TwoWay".
```
```suggestion
The sync state for the open shift request scenario. Valid values are "Disabled", "FromWfmToShifts", and "TwoWay".
```

JohanFreelancer9 commented on teams/teams-ps/teams/New-CsTeamsShiftsConnectionInstance.md:
````diff
@@ -188,74 +155,153 @@ Accept pipeline input: False
+```
 
+### -SyncScenarioShift
+The sync state for the shift scenario. Valid values are "Disabled", "FromWfmToShifts" or "TwoWay".
````
```suggestion
The sync state for the shift scenario. Valid values are "Disabled", "FromWfmToShifts", and "TwoWay".
```

JohanFreelancer9 commented on teams/teams-ps/teams/New-CsTeamsShiftsConnectionInstance.md:
````diff
@@ -188,74 +155,153 @@ Accept pipeline input: False
+```
+
+### -SyncScenarioTimeCard
+The sync state for the time card scenario. Valid values are "Disabled", "FromWfmToShifts" or "TwoWay".
````
```suggestion
The sync state for the time card scenario. Valid values are "Disabled", "FromWfmToShifts", and "TwoWay".
```

JohanFreelancer9 commented on teams/teams-ps/teams/New-CsTeamsShiftsConnectionInstance.md:
```diff
@@ -188,74 +155,153 @@ Accept pipeline input: False
-### -EnabledWfiScenario
-The WFI enabled scenarios that are synced from Shifts in MS Teams to WFM system.
+### -SyncScenarioTimeOff
+The sync state for the time off scenario. Valid values are "Disabled", "FromWfmToShifts" or "TwoWay".
```
```suggestion
The sync state for the time off scenario. Valid values are "Disabled", "FromWfmToShifts", and "TwoWay".
```

JohanFreelancer9 commented on teams/teams-ps/teams/New-CsTeamsShiftsConnectionInstance.md:
````diff
@@ -188,74 +155,153 @@ Accept pipeline input: False
+```
 
+### -SyncScenarioUserShiftPreference
+The sync state for the user shift preferences scenario. Valid values are "Disabled", "FromWfmToShifts" or "TwoWay".
````
```suggestion
The sync state for the user shift preferences scenario. Valid values are "Disabled", "FromWfmToShifts" or "TwoWay".
```

JohanFreelancer9 commented on teams/teams-ps/teams/Set-CsTeamsShiftsConnection.md:
````diff
@@ -0,0 +1,393 @@
+
+```
+
+Updates the instance with the specified -ConnectionId. Returns the object of updated connection.
````
```suggestion
Updates the instance with the specified -ConnectionId. Returns the object of the updated connection.
```

JohanFreelancer9 commented on teams/teams-ps/teams/Set-CsTeamsShiftsConnection.md:
```diff
@@ -0,0 +1,393 @@
+
+Updates the instance with the specified -ConnectionId. Returns the object of updated connection.
+
+In case of error, we can capture the error response as following:
```
```suggestion
In case of an error, you can capture the error response as follows:
```

JohanFreelancer9 commented on teams/teams-ps/teams/Set-CsTeamsShiftsConnection.md:
````diff
@@ -0,0 +1,393 @@
+
+```
+
+Updates the instance with the specified -ConnectionId. Returns the object of updated connection.
````
```suggestion
Updates the instance with the specified -ConnectionId. Returns the object of the updated connection.
```

JohanFreelancer9 commented on teams/teams-ps/teams/Set-CsTeamsShiftsConnection.md:
```diff
@@ -0,0 +1,393 @@
+
+Updates the instance with the specified -ConnectionId. Returns the object of updated connection.
+
+In case of error, we can capture the error response as following:
```
```suggestion
In case of an error, you can capture the error response as follows:
```

JohanFreelancer9 commented on teams/teams-ps/teams/Set-CsTeamsShiftsConnection.md:
````diff
@@ -0,0 +1,393 @@
+```
+
+### -ConnectionId
+The WFM connection id for the instance. This can be retrieved by running [Get-CsTeamsShiftsConnection](Get-CsTeamsShiftsConnection.md).
````
```suggestion
The WFM connection ID for the instance. This can be retrieved by running [Get-CsTeamsShiftsConnection](Get-CsTeamsShiftsConnection.md).
```

JohanFreelancer9 commented on teams/teams-ps/teams/Set-CsTeamsShiftsConnectionInstance.md:
````diff
@@ -18,163 +18,84 @@ This cmdlet updates a Shifts connection instance.
 
+```
 
 Updates the instance with the specified -ConnectorInstanceId. Returns the object of updated connector instance.
````
```suggestion
Updates the instance with the specified -ConnectorInstanceId. Returns the object of the updated connector instance.
```

JohanFreelancer9 commented on teams/teams-ps/teams/Set-CsTeamsShiftsConnectionInstance.md:
````diff
@@ -241,97 +158,174 @@ Gets or sets the list of connector admin email addresses.
+```
 
+### -ConnectionId
+Gets or sets the WFM connection id for the new instance. This can be retrieved by running [Get-CsTeamsShiftsConnection](Get-CsTeamsShiftsConnection.md).
````
```suggestion
Gets or sets the WFM connection ID for the new instance. This can be retrieved by running [Get-CsTeamsShiftsConnection](Get-CsTeamsShiftsConnection.md).
```

JohanFreelancer9 commented on teams/teams-ps/teams/Set-CsTeamsShiftsConnectionInstance.md:
```diff
@@ -241,97 +158,174 @@ Gets or sets the list of connector admin email addresses.
-### -ConnectorInstanceId
-The connector instance id
+### -DesignatedActorId
+Gets or sets the designated actor id that App acts as for Shifts Graph Api calls.
```
```suggestion
Gets or sets the designated actor ID that App acts as for Shifts Graph API calls.
```

JohanFreelancer9 commented on teams/teams-ps/teams/Set-CsTeamsShiftsConnectionInstance.md:
````diff
@@ -241,97 +158,174 @@ Gets or sets the list of connector admin email addresses.
+```
 
+### -SyncScenarioOfferShiftRequest
+The sync state for the offer shift request scenario. Valid values are "Disabled", "FromWfmToShifts" or "TwoWay".
````
```suggestion
The sync state for the offer shift request scenario. Valid values are "Disabled", "FromWfmToShifts" or "TwoWay".
```
```suggestion
The sync state for the offer shift request scenario. Valid values are "Disabled", "FromWfmToShifts", and "TwoWay".
```

JohanFreelancer9 commented on teams/teams-ps/teams/Set-CsTeamsShiftsConnectionInstance.md:
```diff
@@ -241,97 +158,174 @@ Gets or sets the list of connector admin email addresses.
-### -ConnectorSpecificSettings
-The connector specific settings
+### -SyncScenarioOpenShift
+The sync state for the open shift scenario. Valid values are "Disabled", "FromWfmToShifts" or "TwoWay".
```
```suggestion
The sync state for the open shift scenario. Valid values are "Disabled", "FromWfmToShifts", and "TwoWay".
```

JohanFreelancer9 commented on teams/teams-ps/teams/Set-CsTeamsShiftsConnectionInstance.md:
````diff
@@ -241,97 +158,174 @@ Gets or sets the list of connector admin email addresses.
+```
+
+### -SyncScenarioOpenShiftRequest
+The sync state for the open shift request scenario. Valid values are "Disabled", "FromWfmToShifts" or "TwoWay".
````
```suggestion
The sync state for the open shift request scenario. Valid values are "Disabled", "FromWfmToShifts", and "TwoWay".
```

JohanFreelancer9 commented on teams/teams-ps/teams/Set-CsTeamsShiftsConnectionInstance.md:
```diff
@@ -241,97 +158,174 @@ Gets or sets the list of connector admin email addresses.
-### -DesignatedActorId
-Gets or sets the designated actor id that App acts as for Shifts Graph Api calls.
+### -SyncScenarioShift
+The sync state for the shift scenario. Valid values are "Disabled", "FromWfmToShifts" or "TwoWay".
```
```suggestion
The sync state for the shift scenario. Valid values are "Disabled", "FromWfmToShifts", and "TwoWay".
```

JohanFreelancer9 commented on teams/teams-ps/teams/Set-CsTeamsShiftsConnectionInstance.md:
````diff
@@ -241,97 +158,174 @@ Gets or sets the list of connector admin email addresses.
+```
+
+### -SyncScenarioSwapRequest
+The sync state for the shift swap request scenario. Valid values are "Disabled", "FromWfmToShifts" or "TwoWay".
````
```suggestion
The sync state for the shift swap request scenario. Valid values are "Disabled", "FromWfmToShifts", and "TwoWay".
```

JohanFreelancer9 commented on teams/teams-ps/teams/Set-CsTeamsShiftsConnectionInstance.md:
```diff
@@ -241,97 +158,174 @@ Gets or sets the list of connector admin email addresses.
-### -EnabledConnectorScenario
-The connector enabled scenarios that are synced from WFM system to Shifts in MS Teams.
+### -SyncScenarioTimeCard
+The sync state for the time card scenario. Valid values are "Disabled", "FromWfmToShifts" or "TwoWay".
```
```suggestion
The sync state for the time card scenario. Valid values are "Disabled", "FromWfmToShifts", and "TwoWay".
```

JohanFreelancer9 commented on teams/teams-ps/teams/Set-CsTeamsShiftsConnectionInstance.md:
````diff
@@ -241,97 +158,174 @@ Gets or sets the list of connector admin email addresses.
+```
+
+### -SyncScenarioTimeOff
+The sync state for the time off scenario. Valid values are "Disabled", "FromWfmToShifts" or "TwoWay".
````
```suggestion
The sync state for the time off scenario. Valid values are "Disabled", "FromWfmToShifts", and "TwoWay".
```

JohanFreelancer9 commented on teams/teams-ps/teams/Set-CsTeamsShiftsConnectionInstance.md:
```diff
@@ -241,97 +158,174 @@ Gets or sets the list of connector admin email addresses.
-### -EnabledWfiScenario
-The WFI enabled scenarios that are synced from Shifts in MS Teams to WFM system.
+### -SyncScenarioTimeOffRequest
+The sync state for the time off request scenario. Valid values are "Disabled", "FromWfmToShifts" or "TwoWay".
```
```suggestion
The sync state for the time off request scenario. Valid values are "Disabled", "FromWfmToShifts" and "TwoWay".
```

JohanFreelancer9 commented on teams/teams-ps/teams/Set-CsTeamsShiftsConnectionInstance.md:
````diff
@@ -241,97 +158,174 @@ Gets or sets the list of connector admin email addresses.
+```
+
+### -SyncScenarioUserShiftPreference
+The sync state for the user shift preferences scenario. Valid values are "Disabled", "FromWfmToShifts" or "TwoWay".
````
```suggestion
The sync state for the user shift preferences scenario. Valid values are "Disabled", "FromWfmToShifts", and "TwoWay".
```

JohanFreelancer9 commented on teams/teams-ps/teams/Update-CsTeamsShiftsConnectionInstance.md:
```diff
@@ -216,68 +151,147 @@ The connector instance ID.
-### -ConnectorSpecificSettings
-The connector-specific settings.
+### -SyncScenarioOfferShiftRequest
+The sync state for the offer shift request scenario. Valid values are "Disabled", "FromWfmToShifts" or "TwoWay".
```
```suggestion
The sync state for the offer shift request scenario. Valid values are "Disabled", "FromWfmToShifts", and "TwoWay".
```

JohanFreelancer9 commented on teams/teams-ps/teams/Update-CsTeamsShiftsConnectionInstance.md:
```diff
@@ -216,68 +151,147 @@ The connector instance ID.
 
-Required: False
+### -SyncScenarioOpenShift
+The sync state for the open shift scenario. Valid values are "Disabled", "FromWfmToShifts" or "TwoWay".
```
```suggestion
The sync state for the open shift scenario. Valid values are "Disabled", "FromWfmToShifts" and "TwoWay".
```

JohanFreelancer9 commented on teams/teams-ps/teams/Update-CsTeamsShiftsConnectionInstance.md:
```diff
@@ -216,68 +151,147 @@ The connector instance ID.
-### -DesignatedActorId
-The designated actor ID that App acts as for Shifts Graph API calls.
+### -SyncScenarioOpenShiftRequest
+The sync state for the open shift request scenario. Valid values are "Disabled", "FromWfmToShifts" or "TwoWay".
```
```suggestion
The sync state for the open shift request scenario. Valid values are "Disabled", "FromWfmToShifts" and "TwoWay".
```

JohanFreelancer9 commented on teams/teams-ps/teams/Update-CsTeamsShiftsConnectionInstance.md:
```diff
@@ -216,68 +151,147 @@ The connector instance ID.
 
-Required: False
+### -SyncScenarioShift
+The sync state for the shift scenario. Valid values are "Disabled", "FromWfmToShifts" or "TwoWay".
```
```suggestion
The sync state for the shift scenario. Valid values are "Disabled", "FromWfmToShifts", and "TwoWay".
```

JohanFreelancer9 commented on teams/teams-ps/teams/Update-CsTeamsShiftsConnectionInstance.md:
```diff
@@ -216,68 +151,147 @@ The connector instance ID.
-### -EnabledConnectorScenario
-The connector-enabled scenarios that are synced from the WFM system to Shifts in Microsoft Teams.
+### -SyncScenarioSwapRequest
+The sync state for the shift swap request scenario. Valid values are "Disabled", "FromWfmToShifts" or "TwoWay".
```
```suggestion
The sync state for the shift swap request scenario. Valid values are "Disabled", "FromWfmToShifts", and "TwoWay".
```

JohanFreelancer9 commented on teams/teams-ps/teams/Update-CsTeamsShiftsConnectionInstance.md:
```diff
@@ -216,68 +151,147 @@ The connector instance ID.
 
-Required: False
+### -SyncScenarioTimeCard
+The sync state for the time card scenario. Valid values are "Disabled", "FromWfmToShifts" or "TwoWay".
```
```suggestion
The sync state for the time card scenario. Valid values are "Disabled", "FromWfmToShifts", and "TwoWay".
```

JohanFreelancer9 commented on teams/teams-ps/teams/Update-CsTeamsShiftsConnectionInstance.md:
```diff
@@ -216,68 +151,147 @@ The connector instance ID.
-### -EnabledWfiScenario
-The WFI-enabled scenarios that are synced from Shifts in Microsoft Teams to the WFM system.
+### -SyncScenarioTimeOff
+The sync state for the time off scenario. Valid values are "Disabled", "FromWfmToShifts" or "TwoWay".
```
```suggestion
The sync state for the time off scenario. Valid values are "Disabled", "FromWfmToShifts", and "TwoWay".
```

JohanFreelancer9 commented on teams/teams-ps/teams/Update-CsTeamsShiftsConnectionInstance.md:
````diff
@@ -216,68 +151,147 @@ The connector instance ID.
+```
+
+### -SyncScenarioUserShiftPreference
+The sync state for the user shift preferences scenario. Valid values are "Disabled", "FromWfmToShifts" or "TwoWay".
````
```suggestion
The sync state for the user shift preferences scenario. Valid values are "Disabled", "FromWfmToShifts", and "TwoWay".
```

JohanFreelancer9 reviewed (changes requested):
Copy review - @Shane-Malone, kindly commit the suggested changes. Many thanks.

yogkumgit commented:
@get-itips

yogkumgit commented:
@tiburd Please merge this PR. Thanks

Shane-Malone commented:
> @Shane-Malone Hi, I changed this PR to a draft; I believe it should not be merged until the new and updated cmdlets are available. When they are available, please @ mention me in this PR, so I can review and merge (if applicable). Thank you!

@tiburd I believe that the get-help details are generated for cmdlets using the docs in this repo. These docs need to be merged prior to releasing the cmdlets to facilitate this.

traya1 commented on .openpublishing.redirection.json:
```diff
@@ -6929,6 +6929,11 @@
+    },
+    {
+      "source_path": "teams/virtual-folder/teams/New-CsTeamsShiftsConnectionTeamMap.md",
+      "redirect_url": "powershell/module/teams",
```
```suggestion
      "redirect_url": "/powershell/module/teams",
```
Site relative links must start with a slash

traya1 commented:
@Shane-Malone - Who, within the Magic org, are you collaborating with in regard to these updates?

traya1 commented on .openpublishing.redirection.json:
```diff
@@ -6929,6 +6929,11 @@
       "redirect_document_id": false
+    },
+    {
+      "source_path": "teams/teams-ps/teams/New-CsTeamsShiftsConnectionTeamMap.md",
```
```suggestion
      "source_path": "teams/virtual-folder/teams/New-CsTeamsShiftsConnectionTeamMap.md",
```
Setting "source_path" to its previous state as my last commit did not resolve the warning status in the PR

Shane-Malone commented:
> @Shane-Malone - Who, within the Magic org, are you collaborating with in regard to these updates?

@traya1 I am not collaborating with anyone in the magic org as these changes are related to cmdlets in the teams org

Shane-Malone commented:
@tiburd could you remove the draft tag and merge PR. The IC3 team has confirmed that docs are required to be merged before the release of the cmdlets. This is the same as has been done previously.

tiburd commented:
@Shane-Malone 
Hi Shane,

After Thomas, our GitHub admin, approves, I'll proceed.

cc: @traya1

traya1 commented:
The warning needs to be resolved prior to merging the PR. It's a requirement within the Magic org to not merge PRs with a build status with other than `passed`.

traya1 commented:
[like]  Thomas Raya (Aquent LLC) reacted to your message:
________________________________
From: Shane-Malone ***@***.***>
Sent: Wednesday, April 26, 2023 9:11:45 PM
To: MicrosoftDocs/office-docs-powershell ***@***.***>
Cc: Thomas Raya (Aquent LLC) ***@***.***>; Mention ***@***.***>
Subject: Re: [MicrosoftDocs/office-docs-powershell] Update Teams Shifts Connector Cmdlets to V2 APIs (PR #10741)


@traya1<https://github.com/traya1> the validation is passing now. The source path for MAML to YAML files needed to be the yml file extension.

—
Reply to this email directly, view it on GitHub<https://github.com/MicrosoftDocs/office-docs-powershell/pull/10741#issuecomment-1524051713>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/ANEPBQSMXIZTZXSFGYXP35LXDGFRDANCNFSM6AAAAAAXC2Q35I>.
You are receiving this because you were mentioned.Message ID: ***@***.***>
